htp/htp_util.c

```diff
@@ -2366,14 +2366,17 @@ int htp_treat_response_line_as_body(const uint8_t *data, size_t len) {
     //      Firefox 3.5.x: (?i)^\s*http
     //      IE: (?i)^\s*http\s*/
     //      Safari: ^HTTP/\d+\.\d+\s+\d{3}
+    size_t pos = 0;
 
     if (data == NULL) return 1;
-    if (len < 4) return 1;
+    while ((pos < len) && (htp_is_space(data[pos]) || data[pos] == 0)) pos++;
```

Comment:
whats the logic of the test for the 0 byte?

Reply:
I do not know if it is logic but, according to http-evader, some browsers allow space, tabs and null character before response line.
This is the case in http evader 390

Reply:
Ok thanks.